### ISSUE TITLE
Support for rails 7.1

### ISSUE THREAD

**Issue by securityncompliance** [rubysherpas/paranoia#549]
Today I tried to upgrade to 7.1:

```
I could not find compatible versions

Because paranoia >= 2.5.0 depends on activerecord >= 5.1, < 7.1
  and rails >= 7.1.0 depends on activerecord = 7.1.0,
  paranoia >= 2.5.0 is incompatible with rails >= 7.1.0.
So, because Gemfile depends on rails ~> 7.1.0
  and Gemfile depends on paranoia ~> 2.6,
  version solving has failed.
```

You can however use paranoia with `7.1.0.beta`. Can you please add support for 7.1?

**Comment by tisba** [comment]
See https://github.com/rubysherpas/paranoia/pull/548

**Comment by securityncompliance** [comment]
You guys are fast! thanks, will keep an eye on #548

**Comment by mathieujobin** [comment]
released as v2.6.3
